libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/GenState.hs

```diff
@@ -451,14 +455,15 @@ initStableFields proof = do
                   mDeposited = mDeposited ms <+> keydeposit
                 }
           )
+        modify (\gs -> gs {gsInitialDelegations = Map.insert cred kh $ gsInitialDelegations gs})
   zipWithM_ f credentials hashes
 
 runGenRS :: Reflect era => Proof era -> GenSize -> GenRS era ans -> Gen (ans, GenState era)
-runGenRS proof gsize action = do
-  genenv <- genGenEnv proof gsize
-  let action' = initStableFields proof >> action
-  (ans, state, ()) <- runRWST action' genenv (emptyGenState proof genenv)
-  pure (ans, state)
+runGenRS proof gsize action =
+  do
+    genenv <- genGenEnv proof gsize
+    (ans, state, ()) <- runRWST action genenv (emptyGenState proof genenv)
+    pure (ans, state)
```

Comment:
It looks like suggestions I made in another PR are being reverted somehow: https://github.com/input-output-hk/cardano-ledger/pull/2832#discussion_r886741173

In order to reduce the diff:
```suggestion
runGenRS proof gsize action = do
  genenv <- genGenEnv proof gsize
  (ans, state, ()) <- runRWST action genenv (emptyGenState proof genenv)
  pure (ans, state)
```